components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -3354,6 +3446,36 @@ def listOrphanedAnnotations(self, parent_type, parent_ids, eid=None,
         for e in q.findAllByQuery(sql, p, self.SERVICE_OPTS):
             yield AnnotationWrapper._wrap(self, e)
 
+    def countAnnotations(self, objects):
+        """
+        Get the annotion counts for the given objects
+        """
+
+        obj_type = None
+        obj_ids = []
+        for obj in objects:
+            objtype = obj.split('=')[0]
```

Comment:
sorry for being picky, why that has to be a list of strings `type-id`? In the `views.batch_annotate` you have a dict `objs` https://github.com/dominikl/openmicroscopy/blob/c99b0e5c0e7535326d947cc4f0e04666cf938234/components/tools/OmeroWeb/omeroweb/webclient/views.py#L1969.